fluidefi/fluidefi-caspernet-aggregator-tools
# Deploy parser contract hash

**Summary of change:**

- Previously, the deploy parser class returns undefined for any not found contract hash. Currently, the deploy parser returns a place holder of 'Not Available' for any undefined contract hash. 



**List of tasks:** 

- [x] Add place holder value.


**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `enhancement/deploy-parser-contract-hash`
```bash
git checkout enhancement/deploy-parser-contract-hash
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm deploy-parser test has passed.
- Run example.
